t2.lic

```diff
@@ -70,6 +80,25 @@ class T2
       wait_for_script_to_complete(script_name, action_parts)
     end
   end
+
+  def shutdown 
+    if @shutdown == false
```

Comment:
It's minor, but I have two preferences around this style:
1. not explicitly check `== false` (or `== true`)
2. whenever possible, check the `true` case first

So something like:
```
if @shutdown
    # output it's already set to shutdown
else
    # set @shutdown and output
end
````